src/components/vote/VoteButton.tsx

```diff
@@ -1,24 +1,51 @@
 import { useTranslation } from 'react-i18next';
+import { Contracts } from '@ardenthq/sdk-profiles';
+import { useMemo } from 'react';
 import { Button } from '@/shared/components';
 import { Footer } from '@/shared/components/layout/Footer';
 
 export const VoteButton = ({
-    disabled,
+    delegateAddress,
+    fee: _,
+    votes,
     onClick,
-    type = 'vote',
+    isValid: _2,
 }: {
-    disabled: boolean;
+    delegateAddress?: string;
+    fee: string;
+    votes: Contracts.VoteRegistryItem[];
     onClick: () => void;
-    type?: 'vote' | 'unvote' | 'swap_vote';
+    isValid: boolean;
 }) => {
+    // @TODO: disable if empty fee and form is not valid once fee is added
```

Comment:
Would you mind to create a task for it?

Reply:
@shahin-hq im going to remove that code since is removed on subsequent prs anyway

Reply:
@shahin-hq I didnt added a task because that is already handled on https://github.com/ArdentHQ/arkconnect-extension/pull/379